tools/scoring/join_results.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+if [[ $# -lt 2 ]]; then
+    echo "Usage: $0 base-result.csv [adjunct-result.csv [...]]"
+    exit 1
+fi
+
+bname=$(csvcut $1 -c 2 |tail +2 |head -1 |cut -d '/' -f 1)
+
+(
+cat $1;
+shift
+while [[ $# -ne 0 ]]; do
+    adjname=$(csvcut $1 -c 2|tail +2 |head -1 |cut -d '/' -f1)
+    cat $1 |tail +2 |while read -r line; do
+        echo ${line/${adjname}/${bname}};
```

Comment:
Is this bash-specific?  If so, the script should be marked as bash in the `#!`-line.

Reply:
Or use `sed -e "s/${adjname}/${bname}/"` instead of the while loop.

Reply:
I don't know.  Maybe it's better to be safe than sorry and change to bash.